src/examples/accordions/accordion/AccordionBare.tsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Accordion } from '@zendeskgarden/react-accordions';
+import { Row, Col } from '@zendeskgarden/react-grid';
+
+const Example = () => (
+  <Row justifyContent="center">
+    <Col sm={10}>
+      <Accordion level={3} isBare>
```

Comment:
For correct ARIA heading structure within the page, all examples need to use `level={4}` since they fall underneath `<h3>` headings.